features/api/v1/groups/relationships/owners.feature

```diff
@@ -881,20 +881,20 @@ Feature: Group owners relationship
     And the current account has 2 shared "group-owners"
     And the first "group-owner" has the following attributes:
       """
-      { "groupId": "$groups[0]", "userId": "$users[1]" }
+      { "groupId": "$groups[0]", "userId": "$users[2]" }
       """
     And the second "group-owner" has the following attributes:
       """
-      { "groupId": "$groups[0]", "userId": "$users[2]" }
+      { "groupId": "$groups[0]", "userId": "$users[3]" }
```

Comment:
We really need to separate the `$users` placeholder from `$admins`. Having to constantly reason about what index to pick is frustrating and makes tests confusing.